erpnext_documentation/www/docs/user/manual/en/human-resources/setting-up-tax.md

```diff
@@ -1,34 +1,34 @@
 
-#Income Tax Exemption
-In many countries, especially in India, regulations allow exempting a part (or all) of some type of spendings by individuals from being added to their annual taxable income. Examples of such spendings could be contributions to charitable institutions, amount spent for education of children, specific investments etc. To
+# Income Tax Exemption
+In many countries, especially in India, regulations allow exempting a part (or all) of some type of spendings by individuals from being added to their annual taxable income. Examples of such spendings could be contributions to charitable institutions, the amount spent on the education of children, specific investments, etc. To
 avail the exemption from their taxable income, individuals are required to submit proof of such spendings.
 
-ERPNext allows you to configure Tax Slabs as part of every Payroll Period and the applies tax on each Salary based on projected annual earnings of the employee. Tax thus calculated will be added to the Salary Slip for  deduction Salary Component configured _Variable Based On Taxable Salary_. Employees are required to declare the exemption amount they plan to claim at the end of the fiscal year so that the payroll deductions for tax will be calculated based on the projected annual earnings less the exemption. Employees can declare this through [Employee Tax Exemption Declaration](/docs/user/manual/en/human-resources/employee-tax-exemption-declaration), the total of which will be exempted from annual taxable earnings of the employee.
+ERPNext allows you to configure Income Tax Slabs and the applies to the tax on each Salary based on projected annual earnings of the employee. The tax thus calculated will be added to the Salary Slip for deduction Salary Component configured _Variable Based On Taxable Salary_. Employees are required to declare the exemption amount they plan to claim at the end of the fiscal year so that the payroll deductions for tax will be calculated based on the projected annual earnings less than the exemption. Employees can declare this through [Employee Tax Exemption Declaration](/docs/user/manual/en/human-resources/employee-tax-exemption-declaration), the total of which will be exempted from annual taxable earnings of the employee.
 
 If no declaration is submitted by the employee, the monthly deductions will be calculated without any exemption from the employee's annual earnings. However, if the employee submits a declaration in between the payroll period, from the next payroll onwards the exemption will be applied from the next payroll. Any additional tax collected in earlier payrolls will be adjusted in the last payroll or when using _Deduct Tax For Unsubmitted Tax Exemption Proof_ in Payroll Entry or Salary Slip.
```

Comment:
However, if the employee submits a declaration in between the payroll period, the tax exemption will be applied from the next payroll onwards.